src/style.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  background-color: #fff;
+
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  white-space: nowrap;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  padding: 0 50px;
+  margin: 0;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav-list {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  list-style: none;
+  margin: 0;
+  padding-inline-start: 0;
+}
+
+.nav-link {
+  text-decoration: none;
+  display: inline-block;
+  line-height: 60px;
+  margin-left: 20px;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+  color: #000;

```

Comment:
order css properties by type
https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/